### PR TITLE
Setup travis-ci

### PR DESCRIPTION
**Pull request by dmathieu**
There's no tests to execute.
But this makes sure an updated schema will still allow compiling the code.

@cyberdelia after merging this, you will need to enable travis-ci on this repository.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+language: go
+before_script:
+  - go get github.com/google/go-querystring/query
+  - go get github.com/pborman/uuid
```
Comment (cyberdelia) [inline .travis.yml]: I'm wondering if we should use dep here?

Comment (dmathieu) [reply .travis.yml]: That's indeed a possibility.

Comment (cyberdelia) [reply .travis.yml]: Can you try to see if it's not an hassle with travis-ci? If it's a pain, I'll just merge it as is ❤️

Comment (dmathieu) [reply .travis.yml]: I've successfully vendored the dependencies, tests are passing.

**Pull request merged**